annotations_categories.yaml

```diff
@@ -0,0 +1,26 @@
+allOf:
+  - type: object
+    properties:
+
+  - anyOf:
```

Comment:
I think this implies that the file should be annotations.yaml, not annotations_categories.yaml. It should be supporting multiple variants.